### PR TITLE
#1280 feat/add placeholder for empty pic

### LINKED ISSUE #1280: [FEATURE] SfSToreLocator: add placeholder for empty picture
If the store data does not have any picture, then a black space is shown.

We should remove the black space and add a placeholder when the picture is missing.

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #1280 

# Scope of work
Continue work from https://github.com/DivanteLtd/storefront-ui/pull/1339 
Fix after CR and add class to icon.

# Screenshots of visual changes
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented

### CODE REVIEW

**Review by mohammad-k8 — DISMISSED** [review]
Thanks @AdamPawlinski for completing this.

**Comment by AdamPawlinski** [comment]
@mohammad-k8 No problem :)

**Review by filrak — DISMISSED** [review]

File: packages/shared/styles/components/molecules/SfSelect.scss [hunk 1]
```diff
@@ -21,13 +21,8 @@
     );
     @include font(
       --select-label-font,
-<<<<<<< HEAD
```
Comment (justyna-13) [inline packages/shared/styles/components/molecules/SfSelect.scss:24]: Conflict hasn't been resolved :)

**Review by justyna-13 — CHANGES_REQUESTED** [review]

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue [hunk 2]
```diff
@@ -6,75 +6,73 @@
...
-          <div v-if="address" class="sf-store__address">
-            {{ address }}
+    <div :class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
+      <!-- @slot Use this slot to show media elements -->
+      <slot name="media">
+        <SfImage
+          v-if="picture"
+          :src="picture"
+          :alt="`${name} picture`"
+          :width="82"
```
Comment (mayashavin) [inline packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:16]: `width` should not be a fixed manual code. same with `height`. Maybe improve it?

**Review by mayashavin — APPROVED** [review]

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue [hunk 3]
```diff
@@ -6,75 +6,67 @@
...
-          </slot>
-        </div>
-        <!-- @slot This is the default slot of the component, placed on the right of the picture -->
-        <slot>
-          <div v-if="address" class="sf-store__address">
-            {{ address }}
+    <div :class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
+      <!-- @slot Use this slot to show media elements -->
+      <slot name="media">
+        <SfImage v-if="picture" :src="picture" :alt="`${name} picture`" />
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:12]: it should get string instead of object :)

**Review by justyna-13 — COMMENTED** [review]
good job @AdamPawlinski :slightly_smiling_face:   see my comment :)

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue (hunk 2 above)
Comment (mayashavin) [reply packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue]: yes it is :)

**Review by mayashavin — APPROVED** [review]

**Pull request merged**